radionice/pocetni/lekcije/006_pridjevi_imenski_predikat/006_pridjevi_imenski_predikat.tex

```diff
@@ -0,0 +1,247 @@
+		Neprošlost&-な/-だ&-じゃない\\\hline
+		Prošlost&-だった&-じゃなかった\\\hline
+	\end{tabular}
+
+\footnotetext[1]{Iako, u kolokvijalnom jeziku, だ se na kraju ponekad zna izostaviti.}
+
+	\begin{reibun}
+		\rei{\underline{かんたんな}　もんだい　だ。}{Problem je jednostavan.}
+		\rei{へや　は　\underline{きれい　じゃなかった}。}{Soba nije bila čista.}
+		\rei{\underline{まるい}　め　が　\underline{すき　だ}。}{Sviđaju mi se okrugle oči.}
+	\end{reibun}
+	
+
+\fukudai{の pridjevi}
+
+ の pridjevi su zapravo imenice koje opisuju druge imenice tako da ih se povezuje česticom の. Konjugiraju se kao obične imenice.
+ 
+ 	\vspace{10pt}
+	\begin{tabular}{|l|l|}
+		\hline
+		オレンジ&naranča, narančasto\\\hline
+		みどり&zelenilo, zeleno\\\hline
+		みどりいろ&zelena boja, zelene boje\\\hline
+		むらさき&ljubičasta, ljubičasto\\\hline
+		ピンク&roza, rozo\\\hline
+		いろとりどり&varijacija, višebojno\\\hline
+	\end{tabular}
+	
+	\vspace{10pt}
+	\begin{tabular}{|l||l|l|}
+		\hline
+		\multicolumn{3}{c}{Konjugacije の pridjeva}\\\hline
+		 &Pozitivno&Negativno\\\hline
+		Neprošlost&-の/-だ&-じゃない\\\hline
+		Prošlost&-だった&-じゃなかった\\\hline
+	\end{tabular}
+	
+	\begin{reibun}
+	\rei{もうふ　は　\underline{いろとりどり　じゃなかった}。　\underline{みどり　だった}。}{Deka nije bil šarena. Bila je zelena.}
+	\rei{\underline{みどりいろ}　だった。}{Bila je zelene boje.}
+	\rei{\underline{ピンクの}　はな　が　\underline{すき　じゃない}。}{Ne volim rozo cvijeće.}
+	\end{reibun}
+	
+\fukudai{Pristojni oblici}
+
+
+な i の pridjevi, kada se konjugiraju u pristojnom obliku, konjugiraju se isto kao imenice, dok い pridjevi imaju svoja pravila. U negativnom obliku mogu se koristiti -です i -ます oblik, s time da je -ます oblik formalniji od -です.
+
+	\vspace{10pt}
+	\begin{tabular}{|l||l|l|}
+		\hline
+		\multicolumn{3}{c}{Konjugacije な i の pridjeva u pristojnom obliku}\\\hline
+		 &Pozitivno&Negativno\\\hline
+		Neprošlost&-です&-じゃないです/ -じゃありません\\\hline
+		Prošlost&-でした&-じゃなかったです/ -じゃありませんでした\\\hline
+	\end{tabular}
+	
+		\vspace{10pt}
+	\begin{tabular}{|l||l|l|}
+		\hline
+		\multicolumn{3}{c}{Konjugacije い pridjeva u pristojnom obliku}\\\hline
+		 &Pozitivno&Negativno\\\hline
+		Neprošlost&-です&-(い)くないです/ -(い)くありません\\\hline
+		Prošlost&-(い)かったです&-(い)くなかったです/ -(い)くありませんでした\\\hline
+	\end{tabular}
+	
+\vspace{10pt}
+
+
+Ako želite da zvuči još formalnije, možete じゃ pretvoriti u では(čita se kao でわ).
+
+\newpage
+
+\fukudai{Opisne zamjenice}
+
+Opisne zamjenice su zamjenice koje opisuju prvu imenicu ispred koje se nalaze.
+
+
+Par čestih opisnih zamjenica:
+
+ 	\vspace{10pt}
+	\begin{tabular}{|l|l|l|l|}
+		\hline
+		こんな&ovakvo&そんな&takvo\\\hline
+		あんな&onakvo&いろんな&svakakvo\\\hline
+	\end{tabular}
+
+\dai{Imenski predikat}
+	
+Imenski predikati su rečenice koje opisuju neku imenicu. Te rečenice mogu završavati glagolom ili pridjevom.
+
+	\begin{reibun}
+	\rei{\underline{私が　すき　じゃない}　\underline{りんご}}{jabuke koje ja ne volim}
+	\end{reibun}
+	
+\vspace{10pt}
+	私が　すき　じゃない (koje ja ne volim) je imenski predikat koji opisuje sljedeću imenicu りんご (jabuke). Ako se pitamo: "Kakve su to jabuke?", odgovor je: "Jabuke koje ja ne volim.".
```

Comment:
Kad rečenica završava unutar navodnika, ne piše se interpunkcija izvan navodnika, npr "Ovo je citat." Nije potrebno pisati točku za kraj vanjske rečenice, npr "Ovo je loš citat.".